src/languages/en-US/commands/google.json

```diff
@@ -88,10 +89,10 @@
 			]
 		],
 		"examples": [
-			"Madrid",
-			"Arizona"
+			"New York",
+			"San Francisco"
 		],
-		"reminder": "Temperature is in **Celsius** by default. Use the --imperial or --fahrenheit flag to view it in **Fahrenheit**."
+		"reminder": "The units are in the Imperial System by default. Use the --celsius or --metric flag to view it in the **International System**."
```

Comment:
No? The units are in whatever system the place uses

Reply:
Yes, and it's default for the US.